README.md

```diff
@@ -321,17 +318,17 @@ Other examples are provided in the [examples](examples) directory.
 You need first to *clone* this repository and run:
 
 ```bash
-cd zombienet/javascript
-npm install
-npm run build
+❯ cd zombienet/javascript
+❯ npm i && npm run build
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript
+❯ npm i && npm run zombie -- setup <binaries>
```

Comment:
command represented as recommended in the [comment](https://github.com/paritytech/zombienet/pull/1442#discussion_r1373801175)